response.go

```diff
@@ -69,8 +70,12 @@ func (r *Response) decode(dec interface{}) error {
 	if r.Error != nil {
 		return r.Error
 	}
-
-	err := json.NewDecoder(r.Output).Decode(dec)
+	b, e := ioutil.ReadAll(r.Output)
+	if e != nil {
+		return e
+	}
+	fmt.Println(string(b))
```

Comment:
Probably didn't mean to check this patch in.

Reply:
i reverted it.
i'm sorry to patched it. 
that's only my debug source.